actors/builtin/multisig/multisig_test.go

```diff
@@ -539,6 +539,50 @@ func TestCancel(t *testing.T) {
 			Approved: []addr.Address{anne},
 		})
 	})
+
+	t.Run("fail to cancel transaction from any account after removal", func(t *testing.T) {
+		rt := builder.Build(t)
+
+		actor.constructAndVerify(rt, numApprovals, noUnlockDuration, signers...)
+
+		// anne proposes a transaction ID: 0
+		rt.SetCaller(anne, builtin.AccountActorCodeID)
+		rt.ExpectValidateCallerType(builtin.AccountActorCodeID, builtin.MultisigActorCodeID)
+		actor.propose(rt, chuck, sendValue, fakeMethod, fakeParams)
+		rt.Verify()
+
+		// remove anne - tx creator
+		rt.SetCaller(receiver, builtin.MultisigActorCodeID)
+		rt.ExpectValidateCallerAddr(receiver)
+		actor.removeSigner(rt, anne, true)
+		rt.Verify()
+
+		// anne fails to cancel a transaction - she is not a signer
+		//for some reason I cannot ExpectAbort again
+		//rt.SetCaller(anne, builtin.AccountActorCodeID)
+		//rt.ExpectValidateCallerType(builtin.AccountActorCodeID, builtin.MultisigActorCodeID)
+		//rt.ExpectAbort(exitcode.ErrForbidden, func() {
+		//	actor.cancel(rt, txnID)
+		//})
+		//rt.Verify()
+
+		// bob either fails to cancel a transaction - he is not a proposer
```

Comment:
Anne is removed from signer in MultisigActor state, but is not removed from approvers of created transactions. So, now nobody can cancel the tx since proposer is not a signer anymore. Do you think that removed signer should be removed from all transaction approvers?